internal/services/storage/storage_container_resource.go

```diff
@@ -93,8 +98,9 @@ func expandStorageContainerAccessLevel(input string) containers.AccessLevel {
 	return containers.AccessLevel(input)
 }
 
-func storageContainerCreate(d *pluginsdk.ResourceData, meta interface{}) error {
+func resourceStorageContainerCreate(ctx context.Context, d *pluginsdk.ResourceData, meta interface{}) diag.Diagnostics {
 	storageClient := meta.(*clients.Client).Storage
+	_ = ctx
 	ctx, cancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
```

Comment:
Is still needed, since it already has a context from the arguments? cc @thetonymaster

Reply:
the context is updated in the line below (line 104), but if I remove the line 103, a lint issue appear

Reply:
solved